CHANGES

```diff
@@ -18,7 +18,9 @@
 # Generate C API interface functions including man page notes and error
   checking using Python scripts.
 
-# Generate Fortran mpi_f08 binding using Python scripts.
+# Generate Fortran bindings using Python scripts.
```

Comment:
f90 bindings is still not converted

Reply:
OK, I listed them individually to be clearer.